content/departments/product-engineering/engineering/code-graph/code-insights/processes.md

```diff
@@ -46,6 +46,24 @@ Our current process is as follows:
 
 - If one or more issues that were planned for an iteration are looking to **not get finished** (which includes testing) in the [current iteration](https://github.com/orgs/sourcegraph/projects/200/views/1) (while maintaining sustainable work practices) the assignee **raises this as soon as possible asynchronously** to the team (including the PM and EM), e.g. on the GitHub issue or Slack. These issues then **become _proposed_ issues for the next iteration** (meaning nothing carries over automatically, but we also don't just drop and forget missed issues).
 
+## How we think about issue sizes
+
+When sizing an issue please keep in mind the size includes the entire lifecycle of the issue. The coding, any testing both automated and manual, as well as PR reviews, documentation, and any follow-up needed.
+
+When thinking about the size of an issue we try to stay away from actual time estimates. But generally, 1 and 2 sizes would probably finish in less than a full workday. 3 and 5 probably no more than 2 workdays. 8 and 13 get closer to a full week and beyond that begins to span whole weeks or even months.
+
+While there is no "maximum" number, once we get higher than a 13 in size it will probably prompt some discussion or at least need some explanation in the issue.
+
+### Examples
+
+- 1: An issue that you completely know the entire solution and are 100% confident in how long it will take you. You have no questions and no dependencies on other teammates. There aren't any tests that need writing. Fixing a typo, updating copy, or very minor CSS changes would most likely all be 1s.
```

Comment:
Since the above section specifies 
> we try to stay away from actual time estimates

It might make sense to reframe this point around scope, rather than how long it will take.

```suggestion
- 1: A small issue that you completely know the entire solution and are 100% confident in the scope. You have no questions and no dependencies on other teammates. There aren't any tests that need writing. Fixing a typo, updating copy, or very minor CSS changes would most likely all be 1s.
```